setup.rst

```diff
@@ -99,6 +105,13 @@ Open your browser and navigate to ``http://localhost:8000/``. If everything is
 working, you'll see a welcome page. Later, when you are finished working, stop
 the server by pressing ``Ctrl+C`` from your terminal.
 
+.. note::
+
+    Locally, you can also use the PHP built-in webserver to run Symfony using
+    ``php -S localhost:8000 -t public``. This however does not provide most of
```

Comment:
I don't think that using the PHP built-in server is a good idea. It's very limited and comes with many issues. Nging/Apache is a much better alternative.

Reply:
Alright, I was not sure myself and your comment convinced me it was not a good idea. I've now slightly reworded this section instead (to clarify Nginx/Apache can also be used locally).